recipes/mosartwmpy/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "mosartwmpy" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37ec6a6323227260171c45b6c276cd29cdcfed32b3e0fff02b100c466cd07d3c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - bmipy ==2.0
+    - dask ==2021.5.1
+    - matplotlib-base ==3.4.2
+    - nc-time-axis ==1.2.0
+    - netcdf4 ==1.5.6
+    - numexpr ==2.7.3
+    - numpy ==1.20.3
+    - pandas ==1.2.4
+    - pathvalidate ==2.3.0
+    - psutil ==5.8.0
+    - pyarrow ==4.0.1
+    - python >=3.6,<4.0
+    - python-benedict ==0.24.0
+    - regex ==2021.4.4
+    - requests ==2.25.1
+    - rioxarray ==0.4.2
+    - tqdm ==4.60.0
+    - xarray ==0.18.2
+
+test:
+  imports:
+    - mosartwmpy
+    - mosartwmpy.config
+  commands:
+    - python -m unittest discover mosartwmpy
+
+about:
+  home: https://github.com/IMMM-SFA/mosartwmpy
+  summary: 'Python implementation of MOSART-WM: A water routing and management model'
+  license: BSD-2-Clause
```

Despite what upstream says, this is actually the 3-clause version:
```suggestion
  license: BSD-3-Clause
```

---

updated!